setup.py

```diff
@@ -81,17 +72,19 @@ def do_run(self, builder: str, output_dir: str):
         rmtree(correct_doctrees, ignore_errors=True)
         move(self.doctree_dir, correct_doctrees)
 
+
 cmdclass = {}
 cmdclass["build_sphinx"] = MakeDocs
 
 
 install_requires = [
     'sphinx==2.0.*',
-     'cylc==8.0.*',
+    'cylc-flow==8.0a0',
```

Comment:
I think the version needs to be `==8.0*`.

Cylc Flow 8.0a0 appears to be missing the namespace `cylc.flow` (I guess it was done after that release). So the `python setup build_sphinx` fails complaining about missing module `cylc.flow`.

With `cylc==8.0*` the build passes. Perhaps `cylc==8.*`, or `cylc>8`, etc. Just until we release 8.0a1 with the `cylc.flow` namespace change (I think @matthewrmshin said something about a new alpha release soon... or I might be mistaken, but we could probably cut a new alpha release soon)

Reply:
I think this is a case where we really need tight version coupling as we are trying to document a specific version of Cylc.

Lets fix it with the next alpha?